docs/introduction.rst

```diff
@@ -118,51 +118,72 @@ development and test environment for pywbem. That has a larger set of OS-level
 
-
 The Python environment into which you want to install must have the following
-packages installed:
+Python packages installed:
+
+- setuptools - http://pypi.python.org/pypi/setuptools
+- wheel
+- pip - generally installed with Python 3.x but may be a separate install
+  with Python 2.7 and with Cygwin Python releases.
 
-- Python installation support packages(all platforms):
+Pywbemtools installs the pywbem package which requires a number of OS-level
+packages as documented in the `pywbem installation documentation`_.
+The pywbem package provides the ``pywbem_os_setup.sh/.bat`` scripts that
+install the needed OS-level packages.
 
-  - setuptools - http://pypi.python.org/pypi/setuptools
-  - wheel
-  - pip - generally installed with Python 3.x but may be a separate install
-    with Python 2.7 and with Cygwin Python releases.
+To use these scripts, your system must have the following commands installed:
 
-Other installation support packages:
+* On native windows:
 
-- Native windows:
-    - Chocolatey package manager. The pywbemtools package installation uses
-      Chocolatey to install software required for the pywbemtools installation
-      that is are normally available with unix style OSs (ex. make). See
-      https://chocolatey.org/ for the installation instructions for Chocolatey.
-- Windows Cygwin
-    - wget - wget can be installed as part of the Cygwin installation or
-      added with Cygwin package update.
-    - python-devel - Probably named python2-devel / python3-devel
+  - ``choco`` - Chocolatey package manager. The pywbemtools package installation
+    uses Chocolatey to install OS-level software. See https://chocolatey.org/
+    for the installation instructions for Chocolatey.
 
-Pywbemtools installs the pywbem package which has a number of
-prerequisites for installation as documented in the `pywbem installation
-documentation`_. These requirements should be covered by the list above.
+* On all platforms:
+
+  - ``wget`` - Download tool.
 
 
 .. _`Installation with pip`:
 
 Installation with pip
-"""""""""""""""""""""
+^^^^^^^^^^^^^^^^^^^^^
+
+On Linux, OS-X, and UNIX-like environments under Windows, download the
+``pywbem_os_setup.sh`` script and run it to install the needed OS-level
+packages:
+
+.. code-block:: bash
+
+    $ wget -q https://raw.githubusercontent.com/pywbem/pywbemtools/master/pywbem_os_setup.sh
+    $ chmod 755 pywbem_os_setup.sh
+    $ ./pywbem_os_setup.sh
+
+The script uses ``sudo`` under the covers to invoke the OS-level package
+manager for your Linux distribution (e.g. ``yum`` on the RedHat family), so
+the current userid needs to have sudo permission.
+
+Note that for the time being, the pywbem_os_setup.sh script must be downloaded
+from the pywbemtools project. See
+`issue #329 <https://github.com/pywbem/pywbemtools/issues/329>`_.
+
+On native Windows, download the ``pywbem_os_setup.bat`` script and run it to
+install the needed OS-level packages:
+
```

Comment:
how do you get wget.  We are back to chocolatey again

Reply:
wget is stated as a requirement for all platforms. IMO, that is sufficient, i.e. we don't need to explain how to use the package manager of each platform (including native Windows) to get it.